lib/ronin/recon/root.rb

```diff
@@ -23,7 +23,7 @@ module Recon
     # Path to `ronin-recon` root directory.
     #
     # @api private
-    ROOT = File.expand_path(File.join(__dir__,'..','..','..'))
+    ROOT          = File.expand_path(File.join(__dir__,'..','..','..'))
     WORDLISTS_DIR = File.join(ROOT, "data/wordlists")
```

Comment:
`WORDLIST_DIR` needs it's own documentation similar to `ROOT`. Once documentation is added above `WORDLIST_DIR`, the indentation after `ROOT` can be removed.